lib/Payrexx/Communicator.php

```diff
@@ -191,4 +190,15 @@ public function methodAvailable($method)
     {
         return array_key_exists($method, self::$methods);
     }
+
+    /**
+     * Sets the HTTP headers
+     */
+    private function setHttpHeaders(): void
+    {
+        $this->httpHeaders = [
+            'User-Agent' => 'payrexx-php/1.8.0',
+            'x-api-key' => $this->apiSecret,
+        ];
+    }
```

Comment:
I generally like the idea of setting default headers like this. But this implementation overwrites custom headers set via e.g. `$payrexx->setHttpHeaders(['some' => 'header'])` and please rename function to `setDefaultHttpHeaders`.

If easy to do, get the installed payrexx-sdk version and use that as user-agent - otherwise add an sdk-version attribute to `Payrexx.php` which we adjust for each sdk-version.


```suggestion
    /**
     * Sets the default HTTP headers
     */
    private function setDefaultHttpHeaders(): void
    {
        $this->httpHeaders['user-agent'] = 'payrexx-php/1.8.0';
        $this->httpHeaders['x-api-key'] = $this->apiSecret;
    }
```